src/commands/service/doc.ts

```diff
@@ -33,4 +35,23 @@ export default class ServiceDoc extends Command {
     const template = readFileSync(join(__dirname, '..', '..', 'doc.md')).toString()
     return compile(template)(data)
   }
+
+  async saveReadme(servicePath: string, saveFileName: string, markdown: string) {
+    if (saveFileName === undefined) {
+      console.log(markdown)
+      return
+    }
+    let readmeFileName = saveFileName || 'README.md'
+    if (!saveFileName) {
+      let defaultReadmeFileName = readdirSync(servicePath).find(file => {
+        return /^readme(?:.(?:md|txt)+)?$/i.test(file)
+      })
+      if (defaultReadmeFileName) {
+        readmeFileName = defaultReadmeFileName
+      }
+    }
+    writeFile(join(servicePath, readmeFileName), markdown, (err) => {
```

Comment:
should use sync version of `writeFile` function or return a primise.
The callback is throwing an err that nothing will catch..

Reply:
done